### ISSUE TITLE
[Refuses to launch] `#0` not a valid color

### ISSUE THREAD

**Issue by figuld** [rktjmp/shenzhen-solitaire.nvim#3]
Doesn't start the game on the both commands and the lua function.
```
Error executing Lua callback: .../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:32: Vim:E
rror executing Lua callback: ...enzhen-solitaire.nvim/lua/shenzhen-solitaire/ui/view.lua:142: '#0'
is not a valid color
stack traceback:
        [C]: in function 'set_hl'
        ...enzhen-solitaire.nvim/lua/shenzhen-solitaire/ui/view.lua:142: in function 'new'
        ...hen-solitaire.nvim/lua/shenzhen-solitaire/ui/runtime.lua:43: in function <...hen-solitai
re.nvim/lua/shenzhen-solitaire/ui/runtime.lua:31>
stack traceback:
        [builtin#19]: at 0x7f6265cc0640
        [C]: in function 'cmd'
        .../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:32: in function <.../share/nvim
/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:16>
stack traceback:
        [C]: in function 'cmd'
        .../share/nvim/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:32: in function <.../share/nvim
/lazy/lazy.nvim/lua/lazy/core/handler/cmd.lua:16>
```

**Comment by rktjmp** [comment]
Hm, what does 

```
:lua= vim.api.nvim_get_hl_by_name("Normal", true)
```

output? What version of neovim?

**Comment by rktjmp** [comment]
Actually I bet your background color is black (or maybe transparent?). 

If you run

```
:hi Normal guibg=#111111
```

Then

```
:ShenzhenSolitaireNewGame
```

You should be able to play until I write a fix.

**Comment by rktjmp** [comment]
Should be fixed in 3904e3b && 0858c14
